abs.yaml

```diff
@@ -0,0 +1,4 @@
+aggregate_branch: 3.12
+
+channels:
+    - https://staging.continuum.io/prefect/fs/weasel-feedstock/pr1/3891843
```

Comment:
@lorepirri Could you please remove this staging channel after uploading weasel to defaults?

Reply:
@skupr-anaconda yes sure